ShowScheduledEntries script (#24595)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_11_25.md

#### Scripts
##### ShowScheduledEntries

> - Fixed an issue when entry's structure is invalid (Metadata is undefined). Now handles it without crashing.

Reviewer: ```suggestion
- Fixed an issue where the script failed on invalid (undefined) entries in context.
```
